changelog/62346.changed

```diff
@@ -0,0 +1 @@
+Enhance zypper install error messages.
```

Comment:
I think these changes are affecting not only "install" method, right? Maybe a better changelog message would be:

```suggestion
Enhance capture of error messages for Zypper calls in zypperpkg module.
```

Reply:
Absolutely right!